notebooks/8_combine_dataframes.ipynb

```diff
@@ -0,0 +1,1728 @@
+{
```

Comment:
<p>Something is out of order here. Just a few cells above, <code>air_quality</code> had 7 columns, and now it has 12, even though nothing has been done to it in-between.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>I do think this is due to an error in the execution order of the cells, should be ok after rerun.</p>